### PR TITLE
3jhwu9 docker compose

### PR DESCRIPTION
**Pull request by Sckorn**
Closes 3jhwu9

### CODE REVIEW

File: docker/docker-compose/rcbe.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+services:
+  rcbe:
+    environment:
+      - OPER_HOME=/home/oper
+      - HUID
+      - HGID
+      - DISPLAY
+      - QT_X11_NO_MITSHM=1
+
+    image: "dev:${_container_tag}"
```
Comment (radameon) [inline docker/docker-compose/rcbe.yaml:12]: Change this to `rcbe_dev`, re-tag image if needed.

File: docker/scripts/run_docker [hunk 2]
```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# set -x
+export HUID="$(id -u)"
+export HGID="$(id -g)"
+export DISPLAY
+
+usage () { 
+echo "Usage:
+$0 -t <container_tag>	- use specified container tag
```
Comment (radameon) [inline docker/scripts/run_docker:9]: add `l` option description

File: docker/docker-compose/rcbe.yaml (hunk 1 above)
Comment (Sckorn) [reply docker/docker-compose/rcbe.yaml]: Done

File: docker/scripts/run_docker (hunk 2 above)
Comment (Sckorn) [reply docker/scripts/run_docker:10]: Done

**Pull request merged**